misc/python/materialize/output_consistency/input_data/operations/boolean_operations_provider.py

```diff
@@ -25,6 +25,15 @@
 
 BOOLEAN_OPERATION_TYPES: List[DbOperationOrFunction] = []
 
+BOOLEAN_OPERATION_TYPES.append(
```

Comment:
I needed to add a function returning a boolean value to be able to generate nested expressions taking a boolean input.